Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -551,21 +549,18 @@ public void SaveImages(BaseItem item)
 
             CheckDisposed();
 
-            using (WriteLock.Write())
+            using (var connection = GetConnection())
             {
-                using (var connection = CreateConnection())
+                connection.RunInTransaction((Action<IDatabaseConnection>)(db =>
                 {
-                    connection.RunInTransaction(db =>
+                    using (var saveImagesStatement = base.PrepareStatement((IDatabaseConnection)db, (string)"Update TypedBaseItems set Images=@Images where guid=@Id"))
```

Comment:
why explicit cast to `string`?

Reply:
Weird, I didn't touch this. I'll change it back.